content/blog/astradb-haystack-integration/index.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: blog-post
+title: Announcing the Astra DB Haystack Integration
```

Comment:
If we're talking about serverless, the correct product name is "Astra DB Serverless".

Reply:
Astra DB is fine for this purpose